react-frontend/src/components/Nav/navbar.js

```diff
@@ -86,12 +86,12 @@ function NavBar() {
                 Resources
               </Link>
             </li>
-            <li>
+            {/* <li>
```

Comment:
if not needed remove :) if needed leave a comment

Reply:
Will need later!